src/components/banner/mdc-banner/mdc-banner.scss

```diff
@@ -0,0 +1,28 @@
+@import "@limetech/mdc-elevation/mixins";
+@import "@limetech/mdc-theme/mixins";
+@import "@limetech/mdc-rtl/mixins";
+@import "./mixins";
+@import "../../../style/variables";
+
+.mdc-banner {
```

Comment:
again, pretty much my chain of thoughts :D that PR was already in use in their own fork of `material-components-web`. So another option would be to include it in our fork. On the other hand I changed quite a lot from the PR, so whenever it gets merged, migration would be not completely effortless. So we could also just say that it has nothing to do with a material implementation and name it that way

Reply:
If it wouldn't be too much work, we could maybe just change the `mdc-` part of the classes to `lime-` or something? There doesn't need to be much thought behind the prefix, since it's all shadow-dom:ed anyway. Any change that isn't likely to be made by MDC would do.

Reply:
Well, I guess, if we don't import mdc-banner into this component (which we don't since it doesn't exist), I guess those styles, were they to be added in MDC, wouldn't actually affect our banner… but for reduced confusion, and the risk that I'm wrong, changing the prefix is probably a good choice.